### ISSUE TITLE
A small mistake in the algorithm

### ISSUE THREAD

**Issue by connectsoumya** [chyikwei/recommend#1]
Though the program runs without errors, there is a bug in the algorithm. The following will be the corrected ones. Line 123 in [https://github.com/chyikwei/recommend/blob/master/mf/bayesian_matrix_factorization.py](url)

```
 WI_post = inv(inv(self.WI_item) + N * S_bar + \
            np.dot(norm_X_bar, norm_X_bar.T) * \
            (N * self.beta_item) / (self.beta_item + N))
```

Line 157 in [https://github.com/chyikwei/recommend/blob/master/mf/bayesian_matrix_factorization.py](url)

```
 WI_post = inv(inv(self.WI_user) + N * S_bar + \
            np.dot(norm_X_bar, norm_X_bar.T) * \
            (N * self.beta_user) / (self.beta_user + N))
```

**Comment by chyikwei** [comment]
@connectsoumya  thanks! I will check the original literature and the code.

**Comment by chyikwei** [comment]
The error is fixed in #2. thanks!
